cmd/server/main.go

```diff
@@ -227,6 +228,7 @@ func realMain(ctx context.Context) error {
 			sub.Use(rateLimit)
 			sub.Use(requireAuth)
 			sub.Use(requireVerified)
+			sub.Use(requireRealm)
```

Comment:
is this moving MFA after realm selection?

i'm not sure why this is added in this chain

Reply:
Currently registerphone doesn't include a selected realm, so we don't know which realm settings to enforce! Now we have the user first select the realm and then gets here

Reply:
Makes sense to me